### PR TITLE
sys-block/targetcli-fb: new package

### PR DESCRIPTION
**Pull request by diogocp**
targetcli-fb is a fork of targetcli maintained by Red Hat.

Depends on PR #2042

Gentoo-Bug: [590722](https://bugs.gentoo.org/590722)

### CODE REVIEW

File: sys-block/targetcli-fb/targetcli-fb-2.1.43.ebuild [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4} )
```
Comment (SoapGentoo) [inline sys-block/targetcli-fb/targetcli-fb-2.1.43.ebuild]: why no py3.5 support? Please check and add if it works.

**Comment by SoapGentoo** [comment]
Please squash the commits into one.

**Comment by diogocp** [comment]
Added kernel config check.

**Comment by gktrk** [comment]
@gentoo/proxy-maint

**Comment by diogocp** [comment]
Superseded by PR #2042

**Pull request closed without merging**